ref: Update sensitive data management (#2853)
getsentry/sentry-docs: src/platforms/common/data-management/sensitive-data/index.mdx

description: "Learn about filtering or scrubbing sensitive data within the SDK, so that data is not sent with the event. You can also configure server-side scrubbing to ensure the data is not stored."
---

As with any third-party service it’s important to understand what data is being sent to Sentry, and where relevant ensure sensitive data either never reaches the Sentry servers, or at the very least it doesn’t get stored.

> There are two great examples for data scrubbing that every company should think about:

Reviewer: ```suggestion
These are some great examples for data scrubbing that every company should think about:
```

- PII (Personally Identifiable Information) such as a user's name or email address, which post-GDPR should be on every company's mind.
- Authentication credentials, like your AWS password or key.
- Confidential IP (Intellectual Property), such as your favorite color, or your upcoming plans for world domination.

We offer the following options depending on your legal and operational needs:

> - filtering or scrubbing sensitive data within the SDK, so that data is not *sent to* Sentry. Different SDKs have different capabilities, and configuration changes require a redeployment of your application.

Reviewer: I'd suggest a subtle change here:
```suggestion
- filtering or scrubbing sensitive data within the SDK, so that data is *not sent to* Sentry. Different SDKs have different capabilities, and configuration changes require a redeployment of your application.
```

> - [configuring server-side scrubbing](/product/data-management-settings/server-side-scrubbing/) to ensure Sentry does not *store* data. Configuration changes are done in the Sentry UI and apply immediately for new events.

Reviewer: ```suggestion
- [configuring server-side scrubbing](/product/data-management-settings/server-side-scrubbing/) to ensure Sentry does *not store* data. Configuration changes are done in the Sentry UI and apply immediately for new events.
```

> - [running a local Relay](/product/relay/) on your own server between the SDK and Sentry, so that data is not *sent to* Sentry while configuration can still be applied without deploying.

Reviewer: ```suggestion
- [running a local Relay](/product/relay/) on your own server between the SDK and Sentry, so that data is *not sent to* Sentry while configuration can still be applied without deploying.
```
